lib/active_merchant/billing/gateways/airwallex.rb

```diff
@@ -31,15 +31,16 @@ def initialize(options = {})
       end
 
       def purchase(money, card, options = {})
-        payment_intent = create_payment_intent(money, @options)
+        requires!(options, :return_url)
```

Comment:
I think this is the right idea. See my notes in Spreedly channels for ongoing discussion on this pattern.

Reply:
Is this comment in regards to the use of `requires!`?

Reply:
I am 90% sure that it is :D